Rails::Application is abstract, you cannot instantiate it directly

squeemishly opened the pull request:
I upgraded to Rails 5.1.1 and it started throwing me this error message:
_rvm/gems/ruby-2.3.1/gems/railties-5.1.0/lib/rails/railtie.rb:206:in `initialize': Rails::Application is abstract, you cannot instantiate it directly. (RuntimeError))"_ 

I ended up upgrading to ruby 2.3.2 and that fixed the issue.

rafaelfranca committed:
Preparing for 5.1.0.rc1 release

kaspth committed:
Merge pull request #28474 from kamipo/fix_extension_with_darty_target

Fix extension method with dirty target in has_many associations

kaspth committed:
Merge pull request #28406 from chukitow/feature/user-form-with-instead-of-form-for-scaffold

user form with instead of form for in scaffold generator

kaspth committed:
Default to yielding a `form` variable.

More intention revealing and means `f` can go F itself 😋

rafaelfranca committed:
Update the maintenance policy [ci skip]

pixeltrix committed:
Merge pull request #28497 from jhawthorn/unscope_specific_where_value

Fix where_clause#except with specific where value
(cherry picked from commit 81b2cd115356d3a5b3bc28454658101efd16bfad)

rafaelfranca committed:
Merge pull request #28450 from adamgamble/bug_fix/fix_typo_in_actionview

Fix typo in actionview error message in to_form_params helper method

rafaelfranca committed:
Revert "Merge pull request #27775 from domcleal/27774-format-reset"

This reverts commit c6f9f8c28a720ad4ec7cf3613dddfa451d5968e2, reversing
changes made to c309073c7476f50dfb1e796d058580f176101c36.

Reason: This is fixing the behavior in the wrong place. Now the request
path after the request is nil and there is no way to assert that.
Also the test that was added in that PR also fails in 4.2 where the
reporter says it was passing. The reason the bahavior changed between
Rails 4.2 and Rails 5 is that the format in the path is now respected.
The correct way to fix the problem is not doign two requests in the same
controller test and use integrations tests. This change caused a
regression between Rails 5.0.1 and 5.0.2.

claudiob committed:
Move CHANGELOG entry from 5-1 to 5-0 [ci skip]

The entry in the CHANGELOG refers to #25351 which was merged into
Rails 5.0. The behavior did not change in Rails 5.1.

See discussion at https://github.com/rails/rails/commit/2ffa001faa3e9d0295358dfe716b17c54623a086#commitcomment-21410869

rafaelfranca committed:
Fix test that was asserting the wrong thing

rafaelfranca committed:
Fix test assertion

rafaelfranca committed:
Merge pull request #28056 from sngeth/wrap_store_accessors_in_params

Wrap stored accessors in parameters

kaspth committed:
Merge pull request #28510 from claudiob/5-1-stable-changelog-fix

5-1-stable changelog fix

maclover7 committed:
Update bug report templates for 5-1-stable

matthewd committed:
Merge pull request #28516 from maclover7/jm-5-1-templates

Update bug report templates for 5-1-stable

Commit:
Stop complaining about not being able to digest dynamic template partial names and simplify error logging to a single line when not

matthewd committed:
Skip isolated test runs on older rubies

matthewd committed:
Merge pull request #28522 from larskanis/replace-deprecated-pg-constants

[PostgreSQL] Replace deprecated PG constants.

matthewd committed:
Fix a race in ActionCable stream tests

These tests double-stub connection.pubsub, so we need to ensure the
first call's completed before we set up for the second.

matthewd committed:
Merge pull request #28531 from yahonda/suppress_pg_warning

Suppress `WARNING:  nonstandard use of \\ in a string literal` warning

matthewd committed:
Merge pull request #28523 from y-yagi/dont_comment_out_file_watcher_during_rails_upgrade_process

Don't comment out config.file_watcher during Rails upgrade

matthewd committed:
Merge pull request #28530 from kamipo/fix_ci_failure

Fix `assert_logged` failure

rafaelfranca committed:
Merge pull request #28524 from claudiob/comment-api-key

Don't add a dummy API key to every new Rails app

rafaelfranca committed:
Merge pull request #28526 from kamipo/fix_log_subscriber_to_allow_legacy_binds

Fix `LogSubscriber` to allow legacy `binds`

rafaelfranca committed:
Merge pull request #28513 from maclover7/jm-fix-28500

CLI arg "host" has precedence over ENV var "host"

rafaelfranca committed:
Merge pull request #28527 from rst/fix_runner_argv_28515

Correctly reset ARGV for "rails runner `CODE' arg arg arg..."

rafaelfranca committed:
Merge pull request #28528 from domcleal/parseerror-const-deprecation

Change AD::ParamsParser::ParseError deprecation so it can be rescued

rafaelfranca committed:
Merge pull request #28534 from claudiob/list-webpackers

List options for `rails new --webpack=WEBPACK`

rafaelfranca committed:
Merge pull request #28537 from tjschuck/doc_actioncable_reject_unauthed_connx

Document AC::Connection::Authorization#reject_unauthorized_connection

matthewd committed:
Merge pull request #28532 from maclover7/jm-ruby-241

Bump Travis ruby version to 2.4.1

kaspth committed:
Merge pull request #28570 from kamipo/rename_local_current_user_to_verified_user

Rename local variable name `current_user` to `verified_user` [ci skip]

maclover7 committed:
Add WIP Rails 5.1 release notes to documents.yaml

[ci skip]

maclover7 committed:
Upgrade various Rails 5.0 references to 5.1

RC1 is out, `5-1-stable` has been created, figured it was time to start
updating this stuff :)

[ci skip]

maclover7 committed:
Update middleware list

Taken from `bin/rails about` run on a v5.1.0.rc1 application.

[ci skip]

senny committed:
add field ids when generating a scaffold form.

Backport of 27f103fc7e3260efe0b8dde66bf5354f2202ee32

This is a follow up to a6d065e. When using `form_with` you must supply
field ids manually. Since the scaffold generator is using labels we
need to make sure that they are linked up properly.

koic committed:
Use oracle-enhanced 1.8 branch for Rails 5.1 stable branch

eileencodes committed:
Merge pull request #28541 from ota42y/fix/gemfile_generator_fix

ignore system test gems on Gemfile when execute with --skip-test option

tenderlove committed:
Merge pull request #28367 from ptoomey3/ignore-disabled-buttons

Prevent ujs event propagation if element disabled when event chain begins

Fixes #28878
Closes #28837

rafaelfranca committed:
Merge pull request #28578 from koic/use_oracle_enhanced_release_1_8

Use oracle-enhanced 1.8 branch for Rails 5.1 stable branch

rafaelfranca committed:
Merge pull request #28577 from y-yagi/do_not_display_hidden_namespaces_in_destroy_command_help

Do not show hidden namespaces in destroy commnad help

rafaelfranca committed:
Merge pull request #28485 from quantumlicht/fix_migrate_with_empty_version

fix migrate with empty version

rafaelfranca committed:
Merge pull request #27636 from mtsmfm/disable-referential-integrity-without-superuser-privilege-take-2

Use `SET CONSTRAINTS` for `disable_referential_integrity` without superuser privileges (take 2)

rafaelfranca committed:
Merge pull request #28050 from namusyaka/avoid-converting-int-into-float

Avoid converting integer as a string into float

rafaelfranca committed:
Apply the log_level default

Since 4.2 the default log level in production is now debug. We removed
the deprecation in c2e865849beadd99866e521a93d733da7d1b5255
but we don't reflected the default value. We are not appling it.

Closes #28558

rafaelfranca committed:
Merge pull request #28546 from claudiob/drop-j-option

Remove -j (--javascript) option from `rails new`

rafaelfranca committed:
Merge pull request #28557 from y-yagi/remove_unnecessary_files_to_api_applications_when_app_task_task_executed

Remove unnecessary files to API-only Applications when `app:task` task executed

rafaelfranca committed:
Fix the tests to test what they should be testing

With Rack::Test the headers needs to match the `HTTP_` format. The tests
were passing before because they are not asserting the response was a
cache hit.

pixeltrix committed:
Return unmapped timezones from `country_zones`

If a country doesn't exist in the MAPPINGS hash then create a new
`ActiveSupport::Timezone` instance using the supplied timezone id.

Fixes #28431.

pixeltrix committed:
Fix CHANGELOG indentation [ci skip]

pixeltrix committed:
Merge pull request #28514 from y-yagi/follow_up_to_28056

Fix store accessors in parameters test

rafaelfranca committed:
Merge pull request #1 from radiospiel/tickets/26955-listen-paths

No longer listens to dirs outside of the app directory.

pixeltrix committed:
Merge pull request #28590 from y-yagi/remove_unnecessary_javascript_option_check

Remove unnecessary `javascript` option check

eileencodes committed:
Merge pull request #28586 from mtsmfm/override-driver

Make `driven_by` overridable

pixeltrix committed:
Merge pull request #28478 from kamipo/fix_primary_keys_across_multiple_schemas

Fix `primary_keys` across multiple schemas

rafaelfranca committed:
Merge pull request #28592 from htanata/ar_relation_inspect_should_not_load_all_records

Load only needed records on ActiveRecord::Relation#inspect

pixeltrix committed:
Merge pull request #28417 from schneems/schneems/symlink-failures

Raise when using a bad symlink

pixeltrix committed:
Merge pull request #28606 from maclover7/jm-fix-25820

Do not include default response headers for AC::Metal

rafaelfranca committed:
Merge pull request #28617 from javan/rails-ujs/fix-npm-package

Restore rails-ujs.js UMD module support

pixeltrix committed:
Update Ruby versions for Travis CI

eileencodes committed:
Merge pull request #28621 from koic/fix_failed_test_when_running_with_oracle

Fix a failed AR test when running with OracleAdapter

matthewd committed:
Avoid broken faraday 0.12.0 release

matthewd committed:
Revert "Avoid broken faraday 0.12.0 release"

This reverts commit 82a7593e3ac427215a280dae5bd6cd42f9404650.

matthewd committed:
Avoid "can't modify frozen IOError" failures

https://bugs.ruby-lang.org/issues/13239

pixeltrix committed:
Merge pull request #28673 from koic/oracle_does_not_support_index_comment

Ignore AR tests of index comment when using Oracle

matthewd committed:
Work around all the things

jonleighton committed:
Improve logging when Origin header doesn't match

I came up against this while dealing with a misconfigured server. The
browser was setting the Origin header to "https://example.com", but the
Rails app returned "http://example.com" from request.base_url (because
it was failing to detect that HTTPS was used).

This caused verify_authenticity_token to fail, but the message in the
log was "Can't verify CSRF token", which is confusing because the
failure had nothing to do with the CSRF token sent in the request. This
made it very hard to identify the issue, so hopefully this will make it
more obvious for the next person.

rafaelfranca committed:
:scissors:

rafaelfranca committed:
More :scissors:

rafaelfranca committed:
Spaces

matthewd committed:
Remove mentions and instructions for docrails

It's been retired; all contributions now come in via PRs.

matthewd committed:
Drop the "Sanity Check" contributing step

rafaelfranca committed:
Merge pull request #28705 from kamipo/extract_construct_relation_for_exists

Extract `construct_relation_for_exists` in `FinderMethods`

maclover7 committed:
Add scaffolding for different sections in 5.1 release notes

[ci skip]

maclover7 committed:
Add text to "Upgrading to Rails 5.1"

[ci skip]

matthewd committed:
delegate_to_missing doesn't delegate private methods

So we shouldn't claim they're there, even when asked explicitly.

matthewd committed:
delegate_missing_to should fall back to super

bslobodin committed:
exclude ORDER BY clause for exists? (#28699)

rafaelfranca committed:
Merge pull request #28722 from DmytroVasin/rails-ujs-default-url

rails-ujs: missing default url for ajax requests.

matthewd committed:
Merge pull request #28729 from matthewd/dont-freeze-inputs

Don't freeze input strings

rafaelfranca committed:
Merge pull request #28402 from jules2689/action-controller-api-base-hooks

Add `action_controller_base` and `action_controller_api` hooks to mitigate differences between them. Also document Active Support on_load hooks

maclover7 committed:
Move CHANGELOG.md entry from Active Support to Action Pack

Was looking through #28402, and realized the CHANGELOG.md entry is in the wrong
place. Sorry we didn't catch this during code review :cry:

[ci skip]

matthewd committed:
Merge pull request #28740 from matthewd/old-postgres

Use a query that's compatible with PostgreSQL 9.2

matthewd committed:
Merge pull request #28726 from matthewd/transaction-locking

Add comprehensive locking around DB transactions

rafaelfranca committed:
Merge pull request #28744 from y-yagi/make_code_statistics_task_handle_system_tests_properly

Make code statistics task handle system tests properly

rafaelfranca committed:
Merge pull request #28748 from DmytroVasin/rails-ujs-fix-ajax-response-parsing

Fix mistake in JS response parser

matthewd committed:
Lock connection before checking it in

kaspth committed:
Merge pull request #28709 from prathamesh-sonpatki/release-notes-1

Started adding release notes for Rails 5.1 [ci skip]

maclover7 committed:
Revert parts of cad58fb

Changelog links at the top of the components, and lowercased the 'c' in
notable changes. Personally, the styling isn't super important to me, but
it's what we always done in previous guides, so I figured it's better to
keep with what we've always done.

[ci skip]

kaspth committed:
Merge pull request #28631 from y-yagi/fix_28618

Use the config value directly when call `secrets`

kaspth committed:
Let run_secrets_generator handle chdir.

kaspth committed:
Bump the bundled GlobalID version.

kamipo committed:
`cache_key` respects the limit in a relation even if a relation is not loaded

`cache_key` includes the size of a relation. But if a relation is not
loadded, the size is not respected even if a relation has a limit. It
should be respected for consistency.

kaspth committed:
Merge pull request #28685 from rails/smooth-form-with-upgrading

Smooth form with upgrading

pixeltrix committed:
Use more specific check for :format in route path

The current check for whether to add an optional format to the path
is very lax and will match things like `:format_id` where there are
nested resources, e.g:

    resources :formats do
      resources :items
    end

Fix this by using a more restrictive regex pattern that looks for
the patterns `(.:format)`, `.:format` or `/` at the end of the path.
Note that we need to allow for multiple closing parenthesis since
the route may be of this form:

    get "/books(/:action(.:format))", controller: "books"

This probably isn't what's intended since it means that the default
index action route doesn't support a format but we have a test for
it so we need to allow it.

Fixes #28517.

pixeltrix committed:
Merge pull request #28788 from tjschuck/require_as_notifications_in_cache

Explicitly require AS::Notifications in AS::Cache

rafaelfranca committed:
Revert "Merge pull request #28788 from tjschuck/require_as_notifications_in_cache"

This reverts commit 40cc79255dee0e9298269a81f842f0cf62bdaf95.

See 5e2b419779fdb5f0c5566648e2e0fb07429b3540 for the reason

rafaelfranca committed:
Merge pull request #28781 from mtsmfm/sum

Fix Enumerable#sum redefined warning

rafaelfranca committed:
Merge pull request #28790 from tjschuck/require_as_time_in_testing_time_helpers

Explicitly require AS::Time in AS::Testing::TimeHelpers

rafaelfranca committed:
Merge pull request #28791 from y-yagi/date_time_fields_from_list

Remove datetime fields from helper list [ci skip]

pixeltrix committed:
Merge pull request #28793 from kamipo/use_quoted_scope_to_respect_current_database

Use `quoted_scope` rather than `@config[:database]` to respect current database

rafaelfranca committed:
Merge pull request #28734 from rafaelfranca/strong-parameters

Improve the upgrade path of Strong Parameters

rafaelfranca committed:
Merge pull request #28757 from edwardmp/log-actioncable-write-read-errors

Log any write errors originating from the socket

rafaelfranca committed:
Use released sass-rails instead of master version

rafaelfranca committed:
Fix all style violations

rafaelfranca committed:
Merge pull request #28708 from y-yagi/dont_generate_system_test_files

Don't generate system test files if `skip_system_test` option is specified

rafaelfranca committed:
Merge pull request #28733 from kamipo/dont_fallback_to_utf8mb3_after_mysql_8.0.0

Don't fallback to utf8mb3 after MySQL 8.0.0

rafaelfranca committed:
Merge pull request #28814 from halfbyte/fix_error_pages_for_turbolinks

Namespace CSS selectors on error pages

rafaelfranca committed:
Merge pull request #28815 from joshnuss/fix/hash-from-xml-nokogiri

Fixes Hash.from_xml with frozen strings for all engines

rafaelfranca committed:
Merge pull request #28582 from sbull/activejob-error-logging

Add error logging to Active Job

rafaelfranca committed:
Merge pull request #28681 from runephilosof/fix-mysql-grant

Fix quoting in db:create grant all statement.

rafaelfranca committed:
Merge pull request #28640 from kamipo/prevent_double_firing_before_save_callback

Prevent double firing the before save callback of new object when the parent association saved in the callback

rafaelfranca committed:
Merge pull request #28596 from betesh/fix-double-habtm-insert

Fix #24032

rafaelfranca committed:
Merge pull request #28818 from y-yagi/add_require

Add missing require

rafaelfranca committed:
Deprecate raise_on_unfiltered_parameters

rafaelfranca committed:
Preparing for 5.1.0.rc2 release

rafaelfranca committed:
Add test case to make sure we can implicit convert a Parameters to a Hash

rafaelfranca committed:
Configure form_with_generates_remote_forms in its own initializer

This configuration is not present in ActionView::Base so we can't let
the action_view.set_configs initializer set it.

Also add tests to make sure this config works.

Fixes #28824

rafaelfranca committed:
Add note about #26929 in the upgrade guide

This is a small breaking change that we chose to make in 5.1 since the
fix can be done with a search and replace tool.

rafaelfranca committed:
Merge pull request #28829 from y-yagi/follow_up_128b804

Set to `form_with_generates_remote_forms` only when config is explicitly specified

maclover7 committed:
Merge pull request #28841 from pendletons/upgrade-doc-typo

[ci_skip] Fix typo in Rails 5.1 upgrade notes

maclover7 committed:
Add Active Job info to 5.1 release notes

[ci skip]

maclover7 committed:
Finish Action Mailer section in release notes

[ci skip]

maclover7 committed:
Finish Action Cable section in release notes

[ci skip]

maclover7 committed:
Remove unused headings

[ci skip]

maclover7 committed:
Add Active Model info to 5.1 release notes

[ci skip]

maclover7 committed:
Add Railties info to 5.1 release notes

[ci skip]

maclover7 committed:
Add more Railties notable changes

[ci skip]

maclover7 committed:
Remove unused heading

[ci skip]

maclover7 committed:
Small grammar fixes

[ci skip]

maclover7 committed:
Use parameters, not params

Reads better to use the full word.

[ci skip]

maclover7 committed:
Add Action View info to 5.1 release notes

[ci skip]

maclover7 committed:
secrets.yml --> config/secrets.yml

[ci skip]

vipulnsward committed:
Merge pull request #28860 from y-yagi/add_form_with_generates_remote_forms_to_configuring_guide

Add `form_with_generates_remote_forms` to configuring guide [ci skip]

maclover7 committed:
Add missing `action_view` section of config flag

[ci skip]

rafaelfranca committed:
Merge pull request #28871 from aried3r/ar/backport_28776

5-1-stable: `cache_key` respects the limit in a relation even if a relation is not loaded

rafaelfranca committed:
Merge pull request #28828 from kamipo/fix_extending_modules_on_association

Mixin `CollectionProxy::DelegateExtending` after `ClassSpecificRelation`

matthewd committed:
Describe the changes in transactional tests

rafaelfranca committed:
Merge pull request #28839 from y-yagi/fix-yarn-exit-status

Ensure bin/yarn matches the one generated by webpacker

amatsuda committed:
s/broadcasing/broadcasting/

[ci skip]

rafaelfranca committed:
Revert "Merge pull request #27636 from mtsmfm/disable-referential-integrity-without-superuser-privilege-take-2"

This reverts commit 1ccd40ec9842482a58f010d4e3aab2a7c519fec6.

Fixes #28888

Referential actions other than the NO ACTION check cannot be deferred,
even if the constraint is declared deferrable.

claudiob committed:
Merge pull request #28894 from robin850/5-1-notes-fixes

Some improvements to the 5.1 release notes

claudiob committed:
More release notes for Rails 5.1 [ci skip]

@dhh most changes that I mentioned in the talk were already there:
https://speakerdeck.com/claudiob/rails-5-dot-1-upcoming-features

I just added a few that were missing.

maclover7 committed:
Pluralize methods

[ci skip]

rafaelfranca committed:
Merge pull request #28891 from sodabrew/patch-1

Remove mysql2 database adapter default username root

rafaelfranca committed:
Merge pull request #28847 from Edouard-chin/current-page-checkquery-string

Fix `current_page?` regression:

tenderlove committed:
Merge pull request #28897 from rafaelfranca/fix-name-error-error-page

Do not try to encoding the parameters when the controller is not defined

rafaelfranca committed:
Merge pull request #28883 from yahonda/fix28797

PostgreSQL 10 does not convert `CURRENT_DATE` into `('now'::text)::date`

rafaelfranca committed:
Merge pull request #28896 from pschambacher/load_with_shared

Added a shared section to config/database.yml that will be loaded for all envs

rafaelfranca committed:
Merge pull request #28899 from kamipo/restore_fixture_author_addresses

Restore `fixtures :author_addresses`

rafaelfranca committed:
Merge pull request #28906 from georgeclaghorn/fix-belongs-to-default-with-lambda

Evaluate belongs_to :default option against the owner, not the association

rafaelfranca committed:
Also raise error when VERSION is nil

Fix #28905

rafaelfranca committed:
Revert "Merge pull request #28896 from pschambacher/load_with_shared"

This reverts commit 92278a59860c1058bd50304fa59636e5c0e82aae.

This seems to conflict conceptually with #28095 and it is a new feature
after the RC.

rafaelfranca committed:
Merge pull request #28904 from robin850/update-working-with-javascript

Update the "Working with javascript" guide

kaspth committed:
Merge pull request #28844 from kyuden/remove_unnecessary_attributes_of_select_in_form_with

Remove unnecessary `skip_default_ids` and `allow_method_names_outside_object` attributes of select tag in `form_with`

rafaelfranca committed:
Preparing for 5.1.0 release

rafaelfranca committed:
Merge pull request #28911 from prathamesh-sonpatki/ar-release-notes

Added some more release notes for Active Record [ci skip]

rafaelfranca committed:
Merge pull request #28913 from prathamesh-sonpatki/fix-indent

Fix indentation [ci skip]

maclover7 committed:
Small grammar fixes

Will backport to 5-1-stable, since the original PR landed there.

[ci skip]

maclover7 committed:
Small grammar fix

[ci skip]

maclover7 committed:
Small grammar fix

[ci skip]

claudiob committed:
5.1 Release Notes are complete [ci skip]

@prathamesh-sonpatki added the last missing commits in #28911.

eileencodes committed:
Update system test runner docs

It wasn't clear that system tests don't run with the rest of the test
suite and are part of a separate command.

This documents the `test:system` command as well as update the Rails
runner help documentation to make it clearer that system tests are run
via their own command by default.

eileencodes committed:
Use a more realistic use-case for system test generator

It's more likely you'll generate a test for testing users than a users
creates test. This is a better example of a real world test.
Additionally the users create view is unlikely to have the text
"UserCreates" since that is likely generated as well. This is now an
example of visiting the users index.

eileencodes committed:
Clarify documentation for system test arguments

Previously this implied that system tests provided other non-headless
drivers when Selenium is the only driver that uses `:using`,
`:screen_size` or `:options` arguments.

This change clarifies that Selenium is the only non-headless driver.

rafaelfranca committed:
Merge pull request #28920 from eugeneius/require_remove_method

Add missing require for `remove_possible_method`

rafaelfranca committed:
Merge pull request #28924 from y-yagi/fix-release-note

Remove entries included in 5.0 [ci skip]

kamipo committed:
Merge pull request #28934 from y-yagi/fix-note-about-secrets

Fix note about `secrets.yml` [ci skip]

rafaelfranca committed:
Merge pull request #28941 from y-yagi/remove_needless_remove_file

Remove unnecessary `package.json` deletion

rafaelfranca committed:
Merge pull request #28944 from alexcameron89/update_templates

Update the Gem Bug Report Templates to 5.1.0

pixeltrix committed:
Merge pull request #28939 from kirs/maintain_test_schema_friendly

Friendly maintain schema error in test runner

rafaelfranca committed:
Merge pull request #28943 from koshigoe/port-arg-has-precedence-over-env-var

CLI arg `--port` has precedence over env `PORT`

kaspth committed:
Merge pull request #28956 from bdewater/patch-1

Rails secrets is not built on but inspired by Sekrets

rafaelfranca committed:
Merge pull request #28835 from fphilipe/master

Add missing require in ActionMailer::Base

rafaelfranca committed:
Merge pull request #28961 from maclover7/jm-fix-28959

Add type caster to `RuntimeReflection#alias_name`

rafaelfranca committed:
Merge pull request #28978 from riseshia/patch-2

Use target language's document.yaml [ci skip]

rafaelfranca committed:
Merge pull request #28337 from riseshia/patch-1

Fix to use correct path with language on guide

matthewd committed:
Avoid rubygems 2.6.12 for now

It's causing a test to fail, and we're not yet sure what (if anything)
we should be doing differently. See rubygems/rubygems#1911.

matthewd committed:
Merge pull request #28989 from matthewd/quoted_id-deprecation

Clarify deprecation message for #quoted_id

matthewd committed:
Merge pull request #29002 from kamipo/fix_warning_ambiguous_first_argument

Fix `warning: ambiguous first argument`

matthewd committed:
Merge pull request #28240 from kreintjes/fix/autoloading-sti-guide

Correct Autoloading and STI guide (issue #26994)

matthewd committed:
Merge pull request #28995 from jcoyne/update_capybara

Allow capybara minor releases

matthewd committed:
Merge pull request #29022 from y-yagi/allow_to_receive_arbitrary_arguments_in_aggregated_results

Allow to receive arbitrary arguments in `aggregated_results`

tenderlove committed:
Merge pull request #29043 from kamipo/dont_eager_loading_if_unneeded_for_exists

Don't eager loading if unneeded for `FinderMethods#exists?`

tenderlove committed:
Merge pull request #29040 from eugeneius/parameters_delete_block

Pass block in ActionController::Parameters#delete

rafaelfranca committed:
Merge pull request #29034 from peterjm/handle_loops_in_exception_handling

Handle loops in the cause chain in Rescuable#rescue_with_handler

rafaelfranca committed:
Merge pull request #29029 from timolehto/master

rake -T should load development env by default, not test

tenderlove committed:
Bump version for release

tenderlove committed:
Fix the changelogs.

Good job, Aaron

fxn committed:
explain why guide translations need a fork [ci skip]

kaspth committed:
Merge pull request #29086 from mikeycgto/message-encryptor-auth-tag-check

Message encryptor auth tag check

Fixes MessageEncryptor when used in AEAD mode. Specifically, we need to check if the `auth_tag` is nil. This may arise when an AEAD encryptor is used to decrypt a ciphertext generated from a different mode, such as CBC-HMAC. Basically, the number of double dashes will differ and `auth_tag` may be nil in this case.

eileencodes commented:
Something appears to have gone wrong here. What change was this PR supposed to be or was this opened by accident?

rafaelfranca committed:
Fix association documentation

The object inside the scope is of the class that define the association
not the associated class.

[ci skip]

rafaelfranca committed:
Merge pull request #29117 from yahonda/another_29021

Set non 0 value to logger not to be affected by Ruby versions
